### DISCUSSION TITLE
Error while deploying on netlify

### DISCUSSION THREAD

**Discussion by Utkarsh1403** [SVG-Edit/svgedit#865] [Q&A]
hiii Guys, i have made some modification on my locally run svgedit file. after that now i want to host that customize svgedit to netlify but i am facing an issue while deploying. i think i am getting the correct path but still its showing blank screen while clicking on this followed link: https://6396ca99257ac400099d9e3b--monumental-fudge-ad4857.netlify.app/ 
plz help me in this..
Thank you

**Comment by jfhenon** [comment]
Hello, actually, it's not your problem but ours.. The last build (which use svgcanvas from npm) creates an issue with netlify. This also happens on the main site.

I'm looking at this. Sorry for the inconvenience. As a workaround, you could use the Svgedit published on npm (7.3), it should work fine wit your netlify installation.

**Reply by Utkarsh1403** [reply]
thanks for your support

**Reply by Utkarsh1403** [reply]
how can i get all the files of version 7.3 on my local so that i can make my modification.

**Comment by Utkarsh1403** [comment]
2:52:27 PM: build-image version: 4c0c1cadee6a31c9bb8d824514030009c4c05c6a (focal)
2:52:27 PM: build-image tag: v4.15.0
2:52:27 PM: buildbot version: e6d57a43d406fdcaf8ee2df93f280acce2e07259
2:52:27 PM: Fetching cached dependencies
2:52:27 PM: Starting to download cache of 354.8MB
2:52:31 PM: Finished downloading cache in 3.212187619s
2:52:31 PM: Starting to extract cache
2:52:34 PM: Finished extracting cache in 2.950824817s
2:52:34 PM: Finished fetching cache in 6.228520528s
2:52:34 PM: Starting to prepare the repo for build
2:52:34 PM: Preparing Git Reference refs/heads/main
2:52:34 PM: Parsing package.json dependencies
2:52:35 PM: Different build dir detected, going to use the one specified in the Netlify configuration file: '' versus '/' in the Netlify UI
2:52:35 PM: Section completed: initializing
2:52:35 PM: Starting build script
2:52:35 PM: Installing dependencies
2:52:35 PM: Python version set to 2.7
2:52:35 PM: Started restoring cached node version
2:52:36 PM: Finished restoring cached node version
2:52:36 PM: v16.18.1 is already installed.
2:52:36 PM: Now using node v16.18.1 (npm v8.19.2)
2:52:37 PM: Enabling node corepack
2:52:37 PM: Started restoring cached build plugins
2:52:37 PM: Finished restoring cached build plugins
2:52:37 PM: Attempting ruby version 2.7.2, read from environment
2:52:37 PM: Using ruby version 2.7.2
2:52:37 PM: Using PHP version 8.0
2:52:38 PM: npm workspaces detected
2:52:38 PM: Started restoring workspace root node modules
2:52:38 PM: Finished restoring workspace root node modules
2:52:38 PM: Installing NPM modules using NPM version 8.19.2
2:52:43 PM: up to date, audited 1495 packages in 5s
2:52:43 PM: 231 packages are looking for funding
2:52:43 PM:   run `npm fund` for details
2:52:43 PM: 24 vulnerabilities (9 moderate, 10 high, 5 critical)
2:52:43 PM: To address issues that do not require attention, run:
2:52:43 PM:   npm audit fix
2:52:43 PM: To address all issues possible (including breaking changes), run:
2:52:43 PM:   npm audit fix --force
2:52:43 PM: Some issues need review, and may require choosing
2:52:43 PM: a different dependency.
2:52:43 PM: Run `npm audit` for details.
2:52:43 PM: NPM modules installed
2:52:43 PM: Creating package sha
2:52:43 PM: Started restoring cached composer dependencies
2:52:43 PM: Finished restoring cached composer dependencies
2:52:43 PM: No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
2:52:43 PM: Loading composer repositories with package information
2:52:43 PM: Updating dependencies
2:52:43 PM: Nothing to modify in lock file
2:52:43 PM: Writing lock file
2:52:43 PM: Installing dependencies from lock file (including require-dev)
2:52:43 PM: Nothing to install, update or remove
2:52:43 PM: Generating autoload files
2:52:43 PM: Started restoring cached go cache
2:52:43 PM: Finished restoring cached go cache
2:52:44 PM: Installing Go version 1.19.4 (requested 1.19.x)
2:52:49 PM: unset GOOS;
2:52:49 PM: unset GOARCH;
2:52:49 PM: export GOROOT='/opt/buildhome/.gimme/versions/go1.19.4.linux.amd64';
2:52:49 PM: export PATH="/opt/buildhome/.gimme/versions/go1.19.4.linux.amd64/bin:${PATH}";
2:52:49 PM: go version >&2;
2:52:49 PM: export GIMME_ENV="/opt/buildhome/.gimme/env/go1.19.4.linux.amd64.env"
2:52:49 PM: go version go1.19.4 linux/amd64
2:52:49 PM: Detected 0 framework(s)
2:52:49 PM: Installing missing commands
2:52:49 PM: Verify run directory
2:52:50 PM: ​
2:52:50 PM: ────────────────────────────────────────────────────────────────
2:52:50 PM:   Netlify Build                                                 
2:52:50 PM: ────────────────────────────────────────────────────────────────
2:52:50 PM: ​
2:52:50 PM: ❯ Version
2:52:50 PM:   @netlify/build 29.0.3
2:52:50 PM: ​
2:52:50 PM: ❯ Flags
2:52:50 PM:   baseRelDir: true
2:52:50 PM:   buildId: 6396f2cbaf2d9f000835bbc7
2:52:50 PM:   deployId: 6396f2cbaf2d9f000835bbc9
2:52:50 PM: ​
2:52:50 PM: ❯ Current directory
2:52:50 PM:   /opt/build/repo
2:52:50 PM: ​
2:52:50 PM: ❯ Config file
2:52:50 PM:   /opt/build/repo/netlify.toml
2:52:50 PM: ​
2:52:50 PM: ❯ Context
2:52:50 PM:   production
2:52:50 PM: ​
2:52:50 PM: ────────────────────────────────────────────────────────────────
2:52:50 PM:   1. build.command from netlify.toml                            
2:52:50 PM: ────────────────────────────────────────────────────────────────
2:52:50 PM: ​
2:52:50 PM: $ npm run build
2:52:51 PM: > svgedit@7.3.0 build
2:52:51 PM: > rollup -c
2:52:51 PM: 
2:52:51 PM: src/editor/Editor.js → dist/editor/Editor.js, dist/editor/xdomain-Editor.js, dist/editor/iife-Editor.js...
2:52:51 PM: recreating dist
2:53:01 PM: ┌────────────────────────────────────────┐
2:53:01 PM: │                                        │
2:53:01 PM: │   Destination: dist/editor/Editor.js   │
2:53:01 PM: │   Bundle Size:  1.66 MB                │
2:53:01 PM: │   Minified Size:  1.05 MB              │
2:53:01 PM: │   Gzipped Size:  174.85 KB             │
2:53:01 PM: │                                        │
2:53:01 PM: └────────────────────────────────────────┘
2:53:01 PM: ┌────────────────────────────────────────────────┐
2:53:01 PM: │                                                │
2:53:01 PM: │   Destination: dist/editor/xdomain-Editor.js   │
2:53:01 PM: │   Bundle Size:  1.66 MB                        │
2:53:01 PM: │   Minified Size:  1.05 MB                      │
2:53:01 PM: │   Gzipped Size:  174.87 KB                     │
2:53:01 PM: │                                                │
2:53:01 PM: └────────────────────────────────────────────────┘
2:53:01 PM: ┌─────────────────────────────────────────────┐
2:53:01 PM: │                                             │
2:53:01 PM: │   Destination: dist/editor/iife-Editor.js   │
2:53:02 PM: Creating deploy upload records
2:53:01 PM: │   Bundle Size:  1.74 MB                     │
2:53:01 PM: │   Minified Size:  1.03 MB                   │
2:53:01 PM: │   Gzipped Size:  171.59 KB                  │
2:53:01 PM: │                                             │
2:53:01 PM: └─────────────────────────────────────────────┘
2:53:01 PM: (!) Unresolved dependencies
2:53:01 PM: https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
2:53:02 PM: Starting post processing
2:53:01 PM: @svgedit/svgcanvas (imported by "src/editor/Editor.js", "src/editor/EditorStartup.js", "src/editor/MainMenu.js", "src/editor/panels/LeftPanel.js", "src/editor/panels/TopPanel.js", "src/editor/panels/BottomPanel.js", "src/editor/panels/LayersPanel.js", "src/editor/Rulers.js", "src/editor/dialogs/imagePropertiesDialog.js" and "src/editor/components/jgraduate/jQuery.jGraduate.js")
2:53:01 PM: (!) Missing global variable name
2:53:01 PM: https://rollupjs.org/guide/en/#outputglobals
2:53:01 PM: Use "output.globals" to specify browser global variable names corresponding to external modules:
2:53:02 PM: Post processing - HTML
2:53:01 PM: @svgedit/svgcanvas (guessing "SvgCanvas")
2:53:01 PM: created dist/editor/Editor.js, dist/editor/xdomain-Editor.js, dist/editor/iife-Editor.js in 10.1s
2:53:01 PM: 
2:53:01 PM: ./src/editor/extensions/ext-eyedropper/ext-eyedropper.js → dist/editor/extensions/ext-eyedropper...
2:53:02 PM: Post processing - header rules
2:53:01 PM: created dist/editor/extensions/ext-eyedropper in 63ms
2:53:01 PM: 
2:53:01 PM: ./src/editor/extensions/ext-grid/ext-grid.js → dist/editor/extensions/ext-grid...
2:53:02 PM: Post processing - redirect rules
2:53:01 PM: created dist/editor/extensions/ext-grid in 59ms
2:53:01 PM: 
2:53:01 PM: ./src/editor/extensions/ext-helloworld/ext-helloworld.js → dist/editor/extensions/ext-helloworld...
2:53:01 PM: created dist/editor/extensions/ext-helloworld in 51ms
2:53:03 PM: Post processing done
2:53:01 PM: 
2:53:01 PM: ./src/editor/extensions/ext-markers/ext-markers.js → dist/editor/extensions/ext-markers...
2:53:01 PM: created dist/editor/extensions/ext-markers in 42ms
2:53:03 PM: Section completed: postprocessing
2:53:01 PM: 
2:53:01 PM: ./src/editor/extensions/ext-opensave/ext-opensave.js → dist/editor/extensions/ext-opensave...
2:53:01 PM: created dist/editor/extensions/ext-opensave in 121ms
2:53:01 PM: 
2:53:01 PM: ./src/editor/extensions/ext-overview_window/ext-overview_window.js → dist/editor/extensions/ext-overview_window...
2:53:01 PM: created dist/editor/extensions/ext-overview_window in 45ms
2:53:01 PM: 
2:53:01 PM: ./src/editor/extensions/ext-panning/ext-panning.js → dist/editor/extensions/ext-panning...
2:53:02 PM: created dist/editor/extensions/ext-panning in 30ms
2:53:02 PM: 
2:53:02 PM: ./src/editor/extensions/ext-polystar/ext-polystar.js → dist/editor/extensions/ext-polystar...
2:53:02 PM: created dist/editor/extensions/ext-polystar in 84ms
2:53:02 PM: 
2:53:02 PM: ./src/editor/extensions/ext-shapes/ext-shapes.js → dist/editor/extensions/ext-shapes...
2:53:02 PM: created dist/editor/extensions/ext-shapes in 42ms
2:53:02 PM: 
2:53:02 PM: ./src/editor/extensions/ext-storage/ext-storage.js → dist/editor/extensions/ext-storage...
2:53:02 PM: created dist/editor/extensions/ext-storage in 44ms
2:53:02 PM: ​
2:53:02 PM: (build.command completed in 11.4s)
2:53:02 PM: ​
2:53:02 PM: ────────────────────────────────────────────────────────────────
2:53:02 PM:   2. Deploy site                                                
2:53:02 PM: ────────────────────────────────────────────────────────────────
2:53:02 PM: ​
2:53:02 PM: Starting to deploy site from 'dist/editor'
2:53:02 PM: Creating deploy tree 
2:53:02 PM: 0 new files to upload
2:53:02 PM: 0 new functions to upload
2:53:02 PM: Section completed: deploying
2:53:02 PM: Site deploy was successfully initiated
2:53:02 PM: ​
2:53:02 PM: (Deploy site completed in 250ms)
2:53:02 PM: ​
2:53:02 PM: ────────────────────────────────────────────────────────────────
2:53:02 PM:   Netlify Build Complete                                        
2:53:02 PM: ────────────────────────────────────────────────────────────────
2:53:02 PM: ​
2:53:02 PM: (Netlify Build completed in 11.6s)
2:53:02 PM: Caching artifacts
2:53:02 PM: Started saving workspace root node modules
2:53:02 PM: Finished saving workspace root node modules
2:53:02 PM: Started saving build plugins
2:53:02 PM: Finished saving build plugins
2:53:02 PM: Started saving pip cache
2:53:02 PM: Finished saving pip cache
2:53:02 PM: Started saving emacs cask dependencies
2:53:02 PM: Finished saving emacs cask dependencies
2:53:02 PM: Started saving maven dependencies
2:53:02 PM: Finished saving maven dependencies
2:53:02 PM: Started saving boot dependencies
2:53:02 PM: Finished saving boot dependencies
2:53:02 PM: Started saving composer dependencies
2:53:02 PM: Finished saving composer dependencies
2:53:02 PM: Started saving rust rustup cache
2:53:02 PM: Finished saving rust rustup cache
2:53:02 PM: Started saving go dependencies
2:53:02 PM: Finished saving go dependencies
2:53:02 PM: Build script success
2:53:02 PM: Section completed: building
2:53:05 PM: Site is live ✨
2:53:05 PM: Uploading Cache of size 354.8MB
2:53:07 PM: Section completed: cleanup
2:53:07 PM: Finished processing build request in 40.008807202s

This is my logs on netlify.. while deploying..! can you plz help me in this

**Comment by jfhenon** [comment]
This error in your netlify log should now be fixed with the fix #866 

> 2:53:01 PM: (!) Unresolved dependencies

**Reply by Utkarsh1403** [reply]
Thank you soo much for your support.. its working..
